src/query/list_file.c

```diff
@@ -5188,13 +5198,16 @@ qfile_clear_list_cache (THREAD_ENTRY * thread_p, int list_ht_no, bool release)
       /* unhappy condition; if this happens, memory leak will occurrs */
       er_log_debug (ARG_FILE_LINE, "ls_clear_list_cache: failed to delete all entries\n");
     }
-  (void) mht_clear (qfile_List_cache.list_hts[list_ht_no], NULL, NULL);
 
-  /* release assigned memory hash table */
-  if (release)
+  if (qfile_List_cache.list_hts[list_ht_no]->nentries == 0)
```

Comment:
intended to use `qfile_get_list_cache_number_of_entries ()`?

Reply:
I fixed it to use qfile_get_list_cache_number_of_entries.